### PR TITLE
Iht 4351 Move trusts section in post submission PDF into assets section

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -2020,6 +2020,8 @@ pdf.clearanceCertificate.lastParagraph.line1 = This certificate does not itself
 pdf.clearanceCertificate.lastParagraph.line2 = . In particular, the issue of the certificate does not necessarily mean that values have been “ascertained” or that values may be taken as market values for capital gains tax within the provisions of section 274 and paragraph 9, schedule 11, Taxation of Chargeable Gains Act 1992
 pdf.clearanceCertificate.lastParagraph.line3 = . This certificate is not valid in certain circumstances, such as in the case of fraud or failure to disclose material facts or if further tax becomes payable as a result of an instrument of variation – see section 239 (4) Inheritance Tax Act 1984.
```
Comment (dmbeer) [inline conf/messages]: remove blank lines. Keep grouped items together

File: conf/pdf/templates/application/return/iht-return.xsl [hunk 2]
```diff
@@ -8,9 +8,8 @@
     <xsl:include href="pdf/templates/application/return/debts.xsl"/>
     <xsl:include href="pdf/templates/application/return/exemptions.xsl"/>
     <xsl:include href="pdf/templates/application/return/gifts.xsl"/>
-    <xsl:include href="pdf/templates/application/return/trusts.xsl"/>
     <xsl:include href="pdf/templates/application/return/tnrb.xsl"/>
-   <!-- <xsl:include href="pdf/templates/iht-component-templates.xsl"/>-->
+    <!-- <xsl:include href="pdf/templates/iht-component-templates.xsl"/>-->
```
Comment (dmbeer) [inline conf/pdf/templates/application/return/iht-return.xsl]: remove commented out code

Comment (grantrobson) [inline conf/pdf/templates/application/return/iht-return.xsl]: Done

File: conf/messages (hunk 1 above)
Comment (grantrobson) [inline conf/messages]: Done

**Pull request merged**